src/view/Painting.java

```diff
@@ -0,0 +1,38 @@
+package view;
+
+import language.DrawingTask;
+import language.Painter;
+
+import java.awt.Color;
+
+/**
+ * Created by Xiaxing SHI on 21/11/15.
+ */
+public class Painting implements DrawingTask {
```

Comment:
This class is the language we defined. User who prefers writing codes could use this class to draw rather than use directly our Shape objects